packages/core/src/components/AccordionGroup/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './AccordionGroup';
```

Comment:
I think you need to export other components, like Accordion, AccordionContent, etc. from here.

Also, we need to export them from `src/index.tsx`

Reply:
Done